cards/banjo-music/card.md

```diff
@@ -30,6 +30,12 @@ A card about banjos. Updated in MacDown.
 
 ![resonator banjo picture](image-2.jpg)
 
+## Famous People Who Also Play Banjo
+
+- Steve Martin.
+
+![Steve Martin and a banjo](image-3.jpg)
```

Comment:
Needed to do one more screen cast on how to update pull requests. Just busted it out. I think I might leave this PR around a bit in case we need some more tutorials as we get the site up and running with the public.